### DISCUSSION TITLE
Security Onion Console (SOC) provides access to a PCAP interface.

### DISCUSSION THREAD

**Discussion by orionhfast** [Security-Onion-Solutions/securityonion#12853] [General]
![Captura de pantalla 2024-04-24 101508](https://github.com/Security-Onion-Solutions/securityonion/assets/156486345/abf9a9bf-7dea-4c1d-97ec-9c7be96e46be)
I want to add a camera. I don't know what to put in each section of the PCAP. Could someone guide me?

**Comment by migueloomar** [comment]
i have the same problem amigo, sorry for my bad english.

**Comment by TotieBash** [comment]
The bare minimum the you need to fill out are:
The sensor ID:   heimdall-nsm
Filter start time:  2024-04-24 01:00:00.000 -07:00
Filter end time:   2024-04-24 01:10:00.000 -07:00

The above example is from my standalone sensor in my hostname is "heimdall-nsm".  The start time and end time are in RFC3339 date time format in which I am asking to give me PCAP from the system from 2024-04-24 from 1am to 1:10am Pacific Standard time zone.

**Reply by orionhfast** [reply]
Thanks for the information.

**Comment by orionhfast** [comment]
![Captura de pantalla 2024-05-17 103325](https://github.com/Security-Onion-Solutions/securityonion/assets/156486345/245c8b28-fc89-4726-ba90-100f916a0b2d)
